{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "@allure.feature('test_broken')\n    def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\nTest_allure.py:30: Exception"}, "start": 1709625051350, "stop": 1709625051351, "uuid": "1b854732-8040-4361-8dec-91fe2df0b398", "historyId": "1a0939986f1b1a2e245e1bcaef4de2de", "fullName": "TestAllure.Test_allure#test_broken", "labels": [{"name": "feature", "value": "test_broken"}, {"name": "parentSuite", "value": "TestAllure"}, {"name": "suite", "value": "Test_allure"}, {"name": "subSuite"}, {"name": "host", "value": "JM200601001481"}, {"name": "thread", "value": "611604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestAllure.Test_allure"}]}